### PR TITLE
New Serverless Pattern - Aws Translate with Lambda

### PR DESCRIPTION
**Pull request by akkumaha**
*Issue #, if available:*
#1424

*Description of changes:*
This pattern contains a sample AWS Cloud Development Kit (AWS CDK) template that deploys a Lambda Function with an S3 object created trigger to start an Amazon Translate Task and place the results in another S3 bucket.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lambda-s3-translate-cdk/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* AWS Account
+* AWS CLI already configured with Administrator permission
+* [NodeJS 14.x installed](https://nodejs.org/en/download/)
+* CDK v2 installed: See Getting Started With the AWS CDK
+* Python CDK required libraries: (install with pip install -r requirements.txt)
+* Clone this repo!
+
+## Deployment Instructions
+
+1. Within your CDK Python module directory(where all your cdk stacks are located) create a constructs folder and within the constructs folder, create an assets folder.
```
Comment (boyney123) [inline lambda-s3-translate-cdk/README.md:19]: Not sure the deployment instructions are clear. Write the instructions based on deploying this CDK construct here or as a clean checkout of this pattern. 

Have a look at other examples if you get stuck

File: lambda-s3-translate-cdk/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2. Place the `lambda_s3_translate_cdk.py` file in the constructs folder you created and the `file-uploaded-trigger` folder in the assets folder you created.
+3. In your terminal run `CDK Deploy` for the specified stack that uses this construct
+
+### Removing the resources
+
+1. run `CDK Destroy <stack id>` for the specified stack that used this construct
```
Comment (boyney123) [inline lambda-s3-translate-cdk/README.md:25]:
```suggestion
1. run `cdk destroy <stack id>` for the specified stack that used this construct
```

File: lambda-s3-translate-cdk/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        "This pattern contains a sample AWS Cloud Development Kit (AWS CDK) template that deploys a Lambda Function with an S3 object created trigger to start an Amazon Translate Task and place the results in another S3 bucket.",
+        "This pattern deploys one Lambda Function and two S3 Buckets."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/lambda-s3-translate-cdk",
+        "templateURL": "serverless-patterns/lambda-s3-translate-cdk",
+        "projectFolder": "pattern",
+        "templateFile": "assets/lambda-s3-translate-cdk.py"
```
Comment (boyney123) [inline lambda-s3-translate-cdk/example-pattern.json:19]: Think this path is incorrect? The file does not belong in assets folder?

File: lambda-s3-translate-cdk/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      "text": [
+        "Delete the stack: <code>cdk delete</code>."
+      ]
+    },
+    "authors": [
+      {
+        "name": "Kranthi Kiran A",
+        "image": "https://media.licdn.com/dms/image/C5603AQGMzL2iVESvpA/profile-displayphoto-shrink_400_400/0/1662286284377?e=1691020800&v=beta&t=p7litfMASeNBNU-Xeb7bpl5WD8KoWrlVd3Azl3_q1bg",
+        "bio": "Kranthi is an Associate Cloud Developer with Amazon Web Services",
+        "linkedin": "https://www.linkedin.com/in/akkiran003/"
```
Comment (boyney123) [inline lambda-s3-translate-cdk/example-pattern.json:54]:
```suggestion
        "linkedin": "akkiran003"
```

File: lambda-s3-translate-cdk/pattern/lambda_s3_translate_cdk_stack.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                                                                resources=['*']),
+                                                            iam.PolicyStatement(actions=['s3:PutObject'],
+                                                                                resources=[
+                                                                                    self.destination_bucket.bucket_arn+'/*']
+                                                                                ),
+                                                            iam.PolicyStatement(actions=['s3:GetObject',
+                                                                                        's3:ListBucket'],
+                                                                                resources=[
+                                                                                    self.input_bucket.bucket_arn,
+                                                                                    self.input_bucket.bucket_arn + '/*'
```
Comment (boyney123) [inline lambda-s3-translate-cdk/pattern/lambda_s3_translate_cdk_stack.py:56]: Whats with the extra space here in the file formatting? Is this normal?

Comment (akkumaha) [reply lambda-s3-translate-cdk/pattern/lambda_s3_translate_cdk_stack.py:56]: yes, That extra space doesn't really make any change. a + b is same as a+b in python

**Comment by jbesw** [comment]
Thanks for the contribution - this will be live at https://serverlessland.com/patterns/s3-lambda-translate-s3-cdk-python shortly.

**Pull request merged**